packages/web-components/src/image/image.stories.ts

```diff
@@ -98,6 +98,10 @@ export default {
         defaultValue: {
           summary: 'square',
         },
+        type: {
+          summary:
+            'When shape `rounded` is used, default border radius is 4px -- `borderRadiusMedium`. The dev may override the default border radius using one of the following: borderRadiusSmall, borderRadiusLarge, borderRadiusXLarge.',
```

Comment:
I would suggest removing the reference to the actual *token value* and instead just note the token. In the case that the default value changes to 3px or 5px I don't think we want to have to remember to update this. The important note is probably the token itself and not the value.